hono/src/services/outfits.ts

```diff
@@ -33,94 +33,91 @@ const selectOutfitSchema = createSelectSchema(outfits, {
 })
 
 app.get('/', injectDB, async (c) => {
-  return c.json(
-    await c.get('db').query.outfits.findMany({
-      with: {
-        itemsToOutfits: {
-          columns: {
-            itemId: false,
-            outfitId: false,
-          },
-          with: {
-            item: true,
-          },
-          orderBy: (itemsToOutfits, { asc }) => [asc(itemsToOutfits.itemType)],
+  const res = await c.get('db').query.outfits.findMany({
+    with: {
+      itemsToOutfits: {
+        columns: {
+          itemId: false,
+          outfitId: false,
         },
+        with: {
+          item: true,
+        },
+        orderBy: (itemsToOutfits, { asc }) => [asc(itemsToOutfits.itemType)],
       },
-      orderBy: (outfits, { desc }) => [desc(outfits.wearDate)],
-    })
-  )
+    },
+    orderBy: (outfits, { desc }) => [desc(outfits.wearDate)],
+  })
+  return c.json(res)
 })
 
 app.post('/', zValidator('json', insertOutfitSchema), injectDB, async (c) => {
   const body = c.req.valid('json')
+  const res = await c.get('db').transaction(async (tx) => {
+    // Create outfit
+    const newOutfit = (
+      await tx
+        .insert(outfits)
+        .values({
+          ...body,
+          authorUsername: 'rak3rman', // TODO: remove and replace with author integration
```

Comment:
Can this just be an ENV var?

Reply:
This is hardcoded for now awaiting implementation in issue #48